src/Intl/FranceCitiesBundle.php

```diff
@@ -36,6 +36,24 @@ public static function getCity(string $postalCode, string $inseeCode)
         return self::$cities[$postalCode][$inseeCode] ?? null;
     }
 
+    /**
+     * Returns the cities indexed by INSEE code.
+     */
+    public static function getCityByInseeCode(): array
+    {
+        static $citiesByInsee = [];
+
+        if (empty($citiesByInsee)) {
```

Comment:
i don't understand this check. this array will always be empty here ?

Reply:
at the first function call `$citiesByInsee` is empty and so it is processed. But on the second (or more) function it is not processed anymore _Home made cache_ :wink:.